src/styles/blocks/card.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  align-self: center;
+
+  margin: 0 0 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: var(--line-height-big);
+
+  color: var(--color-dark);
+
+  margin: 0 0 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: var(--line-height-small);
+
+  color: var(--color-light);
+
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin: 0 0 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: var(--line-height-small);
+
+  color: var(--color-dark);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin: 0 0 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: var(--line-height-big);
+
+  color: var(--color-light);
+}
+
+.card__price-value {
+  font-weight: 700;
+  line-height: var(--line-height-big);
+
+  color: var(--color-dark);
+}
+
+.card__link {
+  height: 40px;
+
+  background-color: var(--color-button);
+
+  border: 1px solid transparent;
+  border-radius: var(--border-radius-button);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 700;
+  text-decoration: none;
+
+  transition: background-color 0.3s,
+    color 0.3s;
+}
+
+.card__link:hover {
+  background-color: #fff;
+
+  color: var(--color-button);
+
+  border-color: var(--color-button);
+  border-radius: var(--border-radius-button);
```

Comment:
You have border-radius in card__link, so maybe you don't need to add it here and you also could set border-color in card__link, so you wouldn't have to change it here